code/parachain/frame/cosmwasm/src/lib.rs

```diff
@@ -354,6 +362,13 @@ pub mod pallet {
 
 		/// Weight implementation.
 		type WeightInfo: WeightInfo;
+
+		/// account which execute relayer calls IBC exported entry points
+		#[cfg(feature = "ibc")]
+		type IbcRelayerAccount: Get<AccountIdOf<Self>>;
```

Comment:
I doubt this will be a constant, it will probably be provided by the IBC pallet

Reply:
aksing bridge for `IbcHandler.account()`?

Reply:
Probably yes